samples/jaxrs-sample/jaxrs-provider/src/main/resources/microservices/hello/hello.yaml

```diff
@@ -14,7 +14,7 @@
 ## See the License for the specific language governing permissions and
 ## limitations under the License.
 ## ---------------------------------------------------------------------------
-
+#这个契约在本示例中没有任何作用
```

Comment:
This file actually effects how the schema registered in service center. If it is exists, will register this, otherwise will generate a new one based on code. Handy yaml may be different than generated one.